lib/common/output_html.c

```diff
@@ -17,10 +17,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 
-#include <crm/crm.h>
-#include <crm/common/output_internal.h>
+#include <crm_internal.h>
```

Comment:
crm_internal.h should be included before *any* other headers, because it defines _GNU_SOURCE (via portability.h) which affects how the standard library headers behave. That also means this file doesn't need its own definition of that.

Reply:
Fixed, plus added a patch that should have been a part of the curses_formatted_(v)printf stuff.  Somehow, this PR grew an awful lot.